cli/src/factory_ops/unboxing/init

```diff
@@ -245,7 +245,8 @@ echo "Ok." | tee -a ${LOG_FILE}
 
 # Re-configure firewall
 echo -n "Re-configuring firewall................................." | tee -a ${LOG_FILE}
-salt "*" state.apply components.system.firewall.config ${salt_opts}
+salt "*" state.apply components.system.firewall.teardown ${salt_opts}
+salt "*" state.apply components.system.firewall ${salt_opts}
```

Comment:
What about config ? Is this a typo ?

Reply:
It's not a typo. This is a correct command call.

Reply:
> components.system.firewall.config is a subset of components.system.firewall. 
I assume components.system.firewall.teardown is also a subset of components.system.firewall. Is that correct ? 
If Yes, Will components.system.firewall lead to calling of components.system.firewall.teardown as well ?